cf-create-org.sh

```diff
@@ -30,17 +31,22 @@ if ! [[ $SYSTEM_NAME =~ ^[a-zA-Z0-9-]+$ ]]; then
   exit 1
 fi
 
-if ! [[ $NOTE =~ ^[a-zA-Z0-9_-]+$ ]]; then
-  echo "NOTE must contain only letters, numbers, underscores, and hyphens."
+if ! [[ $POP_START =~ ^[0-9]{8}$ ]]; then
+  echo "POP_START should be of the format YYYYMMDD."
   exit 1
 fi
 
-QUOTA_NAME="${AGENCY_NAME}_${IAA_NUMBER}_${NOTE}"
+if ! [[ $POP_END =~ ^[0-9]{8}$ ]]; then
+  echo "POP_END should be of the format YYYYMMDD."
+  exit 1
+fi
+
+QUOTA_NAME="${AGENCY_NAME}_${IAA_NUMBER}_${POP_START}-${POP_END}-01"
```

Comment:
Ummm, not sure how to handle that cleanly, so would like to punt on it if that's ok with you. We can open an issue...?